File: fsharp2/Handler.fs
namespace AwsDotnetFsharp

open Amazon.Lambda.Core
open Amazon.Lambda.APIGatewayEvents

[<assembly:LambdaSerializer(typeof<Amazon.Lambda.Serialization.Json.JsonSerializer>)>]
do ()
    
module Handler =
    let hello(request:APIGatewayProxyRequest, _:ILambdaContext): APIGatewayProxyResponse = {
        StatusCode = 200
        Body = "hello"
    }

File: aws/aws-csharp2.0/Handler.cs
using System;
using Amazon.Lambda.Core;
using Amazon.Lambda.APIGatewayEvents;

[assembly:LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]

namespace AwsDotnetCsharp
{
    public class Handler
    {
        public APIGatewayProxyResponse Hello(APIGatewayProxyRequest apigProxyEvent, ILambdaContext context) => new APIGatewayProxyResponse
        {
            StatusCode = 200,
            Body = "hello"
        };
    }
}